src/test/java/io/github/mivek/parser/AbstractParserTest.java

```diff
@@ -60,19 +63,22 @@ public void testParseWCMultiplePHE() {
     }
 
     @Test
-    public void testParseWCNull() {
+    public void testParseWCDescriptiveIsNotNullButPhenomenonCanBeEmptyAndIntensityCanBeNull() {
         String wcPart = "SH";
 
         WeatherCondition wc = getSut().parseWeatherCondition(wcPart);
 
-        assertNull(wc);
+        assertNotNull(wc);

```

Comment:
This should be reverted, `SH` is always with a precipitation type.

Reply:
Ok, true